### PR TITLE
Bug 1878565: merge device plugin configs that share the same resource name

### PR DESCRIPTION
**Pull request by zshi-redhat**

### CODE REVIEW

File: pkg/controller/sriovnetworknodepolicy/sriovnetworknodepolicy_controller.go [hunk 1]
```diff
@@ -570,8 +570,12 @@ func renderDevicePluginConfigData(pl *sriovnetworkv1.SriovNetworkNodePolicyList)
 		}
 
 		found, i := resourceNameInList(p.Spec.ResourceName, &rcl)
-		netDeviceSelectors := dptypes.NetDeviceSelectors{}
```
Comment (pliurh) [inline pkg/controller/sriovnetworknodepolicy/sriovnetworknodepolicy_controller.go:573]: I think we can keep this unchanged, and not declare later for each condition, can't we?

Comment (zshi-redhat) [reply pkg/controller/sriovnetworknodepolicy/sriovnetworknodepolicy_controller.go:573]: done

**Pull request merged**